<launch>

<!--param name="/robot_name" value="sim01" /-->
<param name="/robot_name" value="dia-miro37" />

<!-- NODE RATE-->
<param name = "rate" value="200" />

<!-- BALL COLOR -->

<arg name="h_limit_min" default="20" doc="Hue: yellow-green start" />
<arg name="h_limit_max" default="40" doc="Hue: yellow-green end" />

<arg name="s_limit_min" default="200" doc="High saturation for vividness" />
<arg name="s_limit_max" default="255" />

<arg name="v_limit_min" default="230" doc="High brightness for visibility" />
<arg name="v_limit_max" default="255" />

<!-- HANDLING COMMANDS -->

<!-- <node name="command_activation" pkg="command_handler" type="command_activation.py" /> -->
<node pkg="cogbiogroup9" type="command_recognition.py" name="command_recognition" output="screen">
    <param name="rosconsole" value="debug"/>
</node>

<!-- SPEECH RECOGNITION -->

<node pkg="cogbiogroup9" type="speech_to_text_node.py" name="speech_to_text_node" output="screen"/>

<!-- ACTIONS: WE NEED TO IMPLEMENT THIS --> 
<include file="launch/actions.launch" />
<!-- <include file="$(find actions)/launch/double_hough_circles.launch" />
<include file="$(find actions)/launch/double_hsv_color_filter.launch" /> -->

</launch>